src/main/java/org/jabref/gui/fieldeditors/contextmenu/EditorMenus.java

```diff
@@ -51,35 +51,35 @@ public static Supplier<List<MenuItem>> getNameMenu(final TextInputControl textIn
     /**
      * The default context menu with a specific menu copying a DOI/ DOI URL.
      *
-     * @param textArea text-area that this menu will be connected to
+     * @param textField text-field that this menu will be connected to
      * @return menu containing items of the default menu and an item for copying a DOI/DOI URL
      */
-    public static Supplier<List<MenuItem>> getDOIMenu(TextArea textArea, DialogService dialogService) {
+    public static Supplier<List<MenuItem>> getDOIMenu(TextField textField, DialogService dialogService) {
         return () -> {
             ActionFactory factory = new ActionFactory();
             ClipBoardManager clipBoardManager = Injector.instantiateModelOrService(ClipBoardManager.class);
-            MenuItem copyDoiMenuItem = factory.createMenuItem(StandardActions.COPY_DOI, new CopyDoiUrlAction(textArea, StandardActions.COPY_DOI, dialogService, clipBoardManager));
-            MenuItem copyDoiUrlMenuItem = factory.createMenuItem(StandardActions.COPY_DOI_URL, new CopyDoiUrlAction(textArea, StandardActions.COPY_DOI_URL, dialogService, clipBoardManager));
+            MenuItem copyDoiMenuItem = factory.createMenuItem(StandardActions.COPY_DOI, new CopyDoiUrlAction(textField, StandardActions.COPY_DOI, dialogService, clipBoardManager));
+            MenuItem copyDoiUrlMenuItem = factory.createMenuItem(StandardActions.COPY_DOI_URL, new CopyDoiUrlAction(textField, StandardActions.COPY_DOI_URL, dialogService, clipBoardManager));
             List<MenuItem> menuItems = new ArrayList<>();
             menuItems.add(copyDoiMenuItem);
             menuItems.add(copyDoiUrlMenuItem);
             menuItems.add(new SeparatorMenuItem());
-            menuItems.addAll(new DefaultMenu(textArea).get());
+            menuItems.addAll(new DefaultMenu(textField).get());
             return menuItems;
         };
     }
 
     /**
      * The default context menu with a specific menu item to cleanup URL.
      *
-     * @param textArea text-area that this menu will be connected to
+     * @param textField text-area that this menu will be connected to
```

Comment:
`text-area` there

Reply:
The diff shows that `textField` is put in here, thus good or me!